ui/src/components/ripple-out/ProgressSection.js

```diff
@@ -27,8 +27,9 @@ import MilestoneProgressBar from './MilestoneProgressBar';
 //   ]
 // }
 
-const ProgressSection = () => {
+const ProgressSection = ({ bodyFontSize }) => {
   const [fundraiseData, setFundraiseData] = useState(null);
+  const [pledeData, setPledgeData] = useState(null);
```

Comment:
typo here, `pledgeData` instead of "plede" haha so need to refactor for the rest too